packages/expo-cli/src/credentials/local/index.ts

```diff
@@ -1 +1,2 @@
 export { default as credentialsJson } from './credentialsJson';
+export { updateLocalCredentialsJsonAsync } from './update';
```

Comment:
What's the benefit of having a file like this? I think we've discussed this many time :D